index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Home</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"/>
+  <link rel="stylesheet" href="index_styles.css">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Rounded:opsz,wght,FILL,GRAD@48,400,0,0" />
+</head>
+<body>
+  <!-- common header -->
+  <header>
+    <ul class="top-nav">
+      <li><i class="fab fa-brands fa-facebook-f"></i></li>
+      <li><i class="fab fa-brands fa-twitter"></i></li>
+      <li>English</li>
+      <li>My Page</li>
+      <li>Logout</li>
+    </ul>
+    <nav>
+      <div id="hamburger-icon">
+        <div class="bar1"></div>
+        <div class="bar2"></div>
+        <div class="bar3"></div>
+      </div>
+      <section class="mobile-menu">
+        <ul class="mobile-menu-background">
+          <li><a href="index.html">
+            <div class="logo-container">
+              <div class="gear"><span class="material-symbols-rounded">settings_suggest</span></div>
+              <div class="large-text" >GhEng</div>
+            </div>
+          </a></li>
+          <li><a href="about.html" class="orange">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+          <li class="campaign"><a href="#" class="orange">Campaign</a></li>
+        </ul>
+      </section>
```

Comment:
- Hey, having a section tag in the nav is not semantically correct. To enforce good use of the semantic tag, since you've used the section here as a container/divider, dividing the nav by adding other contents, you can use the `div` tag instead of a section tag. please, rename the section to a div tag.